examples/hello_world/hello_world.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
```

Comment:
We should not commit the elf

Reply:
I see all 4 binaries commited

Reply:
Sure, my bad, deleted all of them

Reply:
please verify they are not in git history

Reply:
They're not